metron-deployment/packaging/ambari/src/main/resources/common-services/INDEXING/0.2.0BETA/service_advisor.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env ambari-python-wrap
+"""
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import imp
+import os
+import traceback
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+STACKS_DIR = os.path.join(SCRIPT_DIR, '../../../stacks/')
+PARENT_FILE = os.path.join(STACKS_DIR, 'service_advisor.py')
+
+try:
+    with open(PARENT_FILE, 'rb') as fp:
+        service_advisor = imp.load_module('service_advisor', fp, PARENT_FILE, ('.py', 'rb', imp.PY_SOURCE))
+except Exception as e:
+    traceback.print_exc()
+    print("Failed to load parent service_advisor file '{}'".format(PARENT_FILE))
+
+
+class INDEXING020BETAServiceAdvisor(service_advisor.ServiceAdvisor):
```

Comment:
It's an Ambari imposed pain point. From what I can tell, it just looks for that class name via magic. If this actually existed in it's own stack, we could use the Stack advisor. Service advisor is more for adding custom services to another stack, so this is probably closer to what we can do until everything coalesces a little more nicely.

The better way is probably to have this handled by an Ambari mpack (which bundles everything up on it's own), but I'm not sure how nicely Service and Stack advisors play, particularly when an mpack is installed alongside an existing stack.  It's something I don't know enough about to say.

Relevant portion of the [Ambari wiki](https://cwiki.apache.org/confluence/display/AMBARI/How-To+Define+Stacks+and+Services#How-ToDefineStacksandServices-ServiceAdvisor):


"Unlike the Stack-advisor scripts, the service-advisor scripts do not automatically extend the parent service's service-advisor scripts. The service-advisor script needs to explicitly extend their parent's service service-advisor script."

I wouldn't be opposed to dropping the service-advisor for now (especially given the note in the original description).  Especially since I'd like to drop this method anyway in the future, assuming that's feasible.